[IMP] accounting: updated chart of accounts page (#6617)
odoo/documentation: content/applications/finance/accounting/get_started/chart_of_accounts.rst

=================

The **chart of accounts (COA)** is the list of all the accounts used to record financial
transactions in the general ledger of an organization. The chart of accounts can be found under
:menuselection:`Accounting-->Configuration-->Chart of Accounts`.

When browsing your chart of accounts, you can notably filter the accounts by :guilabel:`Code`,

> :guilabel:`Account Name`, or :guilabel:`Type`, but other options are available in the dropdown menu

Reviewer: FYI there's dropdown and drop-down spelling in the same doc, I'd choose just one ;)

Assets, deferred expenses, and deferred revenues automation
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Some **account types** can **automate** the creation of :ref:`assets <assets-automation>` entries,
:ref:`deferred expenses <deferred-expenses-automation>` entries, and

> :ref:`deferred revenues <deferred-revenues-automation>` entries. To **automate** entries, click on

Reviewer: ```suggestion
:ref:`deferred revenues <deferred-revenues-automation>` entries. To **automate** entries, click
```

Removed 'click ON'

:guilabel:`Setup` on an account line and go to the :guilabel:`Automation` tab.

You have three choices for the :guilabel:`Automation` tab:

#. :guilabel:`No`: this is the default value. Nothing happens.
#. :guilabel:`Create in draft`: whenever a transaction is posted on the account, a draft entry is
   created, but not validated. You must first fill out the corresponding form.
#. :guilabel:`Create and validate`: you must also select a :guilabel:`Deferred Expense Model`.
   Whenever a transaction is posted on the account, an entry is created and immediately validated.

> Default taxes

Reviewer: For idiots like me, please specify where to go for this (click 'Setup')

Author: It's specified on line 99

-------------

Select a **default tax** to be applied when this account is chosen for a product sale or purchase.

Tags
----

> Some accounting reports require **tags** to be set on the relevant accounts. To add a tag, click on

Reviewer: ```suggestion
Some accounting reports require **tags** to be set on the relevant accounts. To add a tag, click
```

Removed 'on'

> the :guilabel:`Tags` field and select an existing tag, or :guilabel:`Create` a new one.

Reviewer: ```suggestion
the :guilabel:`Tags` field and select an existing tag or :guilabel:`Create` a new one.
```

Account groups
--------------

**Account groups** are useful to list multiple accounts as *sub-accounts* of a bigger account and
thus consolidate reports such as the **Trial Balance**. By default, groups are handled automatically
based on the code of the group. For example, a new account `131200` is going to be part of the group

> `131000`. You can attribute a specific group in the :guilabel:`Group` field.

Reviewer: ```suggestion
`131000`. You can attribute a specific group to an account in the :guilabel:`Group` field.
```

I would add "to an account" for clarity, but up to you

Create account groups manually
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. note::
   Regular users should not need to create account groups manually. The following section is only
   intended for rare and advanced use cases.

> To create a new account group, :ref:`developer mode <developer-mode>` and head to

Reviewer: ```suggestion
To create a new account group, activate :ref:`developer mode <developer-mode>` and head to
```

I think a verb is missing here

--------------

**Account groups** are useful to list multiple accounts as *sub-accounts* of a bigger account and
thus consolidate reports such as the **Trial Balance**. By default, groups are handled automatically
based on the code of the group. For example, a new account `131200` is going to be part of the group
`131000`. You can attribute a specific group to an account in the :guilabel:`Group` field under
:guilabel:`Setup`.

Create account groups manually
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

.. note::
   Regular users should not need to create account groups manually. The following section is only
   intended for rare and advanced use cases.

To create a new account group, activate :ref:`developer mode <developer-mode>` and head to
:menuselection:`Accounting app-->Configuration-->Account Groups`. Here, create a new group and enter
the :guilabel:`name, code prefix, and company` to which that group account should be available. Note
that you must enter the same code prefix in both :guilabel:`From` and :guilabel:`to` fields.

.. image:: chart_of_accounts/account-groups.png
   :alt: Account groups creation.

To display your **Trial Balance** report with your account groups, go to

> :menuselection:`Accounting-->Reporting-->Trial Balance`, then open the :guilabel:`Options` menu and

Reviewer 2: ```suggestion
:menuselection:`Accounting --> Reporting --> Trial Balance`, then open the :guilabel:`Options` menu and
```

its payment. Therefore, the account used to record credit card payments needs to be configured as
**allowing reconciliation**.

To do so, check the :guilabel:`Allow Reconciliation` box in the account's settings, and
:guilabel:`Save`; or enable the button from the chart of accounts view.

.. image:: chart_of_accounts/chart-of-accounts-reconciliation.png
   :alt: Allow reconciliation for accounts in Odoo Accounting

Deprecated
----------

It is not possible to delete an account once a transaction has been recorded on it. You can make

> them unusable by using the **Deprecated** feature: check the :guilabel:`Deprecated` box in the
> account's settings, and :guilabel:`Save`.

Reviewer 2: ```suggestion
them unusable by checking the :guilabel:`Deprecated` box in the
account's settings.
```
Suggestion to shorten
